data/test_assets/dataset_tests/test_dataset.dataset_config.json

```diff
@@ -0,0 +1,100 @@
+{
+    "stages": {
+        "default_attributes": {
+            "origin":[1.0,2.0,3.0]
+        },
+        "paths": ["stages"],
+        "configs" : [
+        {
+            "original_file": "dataset_test_stage.stage_config.json",
+            "template_handle": "modified_test_stage",
+            "attributes": {
+				"scale":[1,1,1],
+				"gravity":[0,-9.8,0],
+				"margin":0.041,
+				"friction_coefficient": 0.4,
+				"restitution_coefficient": 0.5,
+				"units_to_meters":1.0
+             }
+        },
+        {
+            "template_handle": "new_test_stage",
+            "asset_source_dir": "stages",
+            "attributes": {
+				"render_asset": "dataset_test_stage.glb",
+				"up":[0,-1,0],
+				"scale":[2,2,2],
+				"gravity":[0,9.8,0],
+				"friction_coefficient": 0.35,
+				"restitution_coefficient": 0.25,
+				"units_to_meters":2.0,
+				"requires_lighting":false
+            }
+        }
+        ]
+    },
+    "objects":{
+        "default_attributes": {
+        },
+        "paths": ["objects"],
+        "configs" : [
+        {
+            "original_file": "dataset_test_object1.object_config.json",
+            "template_handle": "modified_test_object1_1_slick_heavy",
+            "attributes": {
+				"friction_coefficient": 0.1,
+				"mass": 1.038
+            }
+        },
+		{
+            "template_handle": "new_test_object3",
+            "asset_source_dir": "objects",
+            "attributes": {
+				"render_asset": "dataset_test_object_3.glb",
+				"friction_coefficient": 0.1,
+				"mass": 1.038
+            }
+        }
+
+        ]
+    },
+    "light_setups":{
+        "default_attributes": {
+        },
+        "paths": ["lights"],
+        "configs" : [
+        {
+            "original_file":"dataset_test_lights.lighting_config.json",
+            "template_handle": "modified_test_lights",
+            "attributes": {
+  			    "lights":{
+					"0": { "position": [1.5,0.1,1.5], "intensity": 2.4, "color": [0.5,1,0.95], "type": "point"},
```

Comment:
May I ask what the relation is between the lights setup here and the lights specified in the  dataset_tests/lights/dataset_test_lights.lighting_config.json?

Reply:
This is a way for an individual to create a new lighting setup ("modified_test_lights"), based on an existing one ("dataset_test_lights.lighting_config.json"), where specified lighting values are overridden - in this case, lights with IDs 0,1 and 11.